src/components/pages/doc.js

```diff
@@ -383,7 +384,10 @@ function Doc({ initialState, PageTemplate }) {
     <Fragment>
       <PageTemplate alerts={alerts}>
         <div className="DocPage">
-          <MetadataTitle text={selectedHistoryRevision ? selectedHistoryRevision.title : doc.title} />
+          <MetadataTitle
+            text={selectedHistoryRevision ? selectedHistoryRevision.title : doc.title}
```

Comment:
μ-thing to shorten this condition: `selectedHistoryRevision?.title || doc.title` ?

Reply:
I think we wrote this back when titles were allowed to be empty string. Because it's not a fallback display in this case: if `selectedHistoryRevision` is defined then its title has to be shown no matter what and never the `doc.title` (as we are in `history` mode). That's why I don't feel too well having the construct with the OR as it would be actually logically "wrong" and potentially misleading.

Reply:
Oh, that's true! Though in theory there should be no empty titles, this fallback could hide a bug. Got it.

Reply:
Oh no, we wrote this later of course, but the same reason applies why I'd like to leave it like it is.